### DISCUSSION TITLE
Get an error running the test sample

### DISCUSSION THREAD

**Discussion by rebol016** [Future-House/paper-qa#104] [Q&A]
"Retrying langchain.chat_models.openai.acompletion_with_retry.<locals>._completion_with_retry in 4.0 seconds as it raised APIConnectionError: Error communicating with OpenAI"

Get the error message above， anyone can help？  Does it has something to do with the version of openai package?
Thanks in advance

**Comment by whitead** [comment]
Just means OpenAI was unavailable at the time.
